rainier-compute/src/main/scala/com/stripe/rainier/compute/Evaluator.scala

```diff
@@ -2,6 +2,8 @@ package com.stripe.rainier.compute
 
 class Evaluator(var cache: Map[Real, Double]) extends Numeric[Real] {
 
+  def parseString(str: String): Option[Real] = ???
```

Comment:
This is now required by `Numeric`, do we want to just use the underlying `parseString` for `Double` and then create a `Real` from it?